src/examples/recognised-overseas-pension-schemes-notification-list/example/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+  layout: layout-example.njk
+---
+
+<h3 class="govuk-heading-l">Austria</h3>
```

Comment:
Should this be an `<h3>`? Entirely possible that it should be due to the fact that this is a page fragment (I'm guessing) rather than a full page.

Reply:
Seems ok to me in context: https://www.gov.uk/guidance/check-the-recognised-overseas-pension-schemes-notification-list